ui/src/components/helpers/WelcomeTutorialFunctions.ts

```diff
@@ -0,0 +1,33 @@
+const tutorialKey = "tutorial_timestamp";
+
+function setTimeStamp() {
+  const dateItem = {
+    value: "value",
+    expiry: new Date().getTime(),
+  };
+  localStorage.setItem(tutorialKey, JSON.stringify(dateItem));
+}
+
+function checkIfValidTimestamp(totalTimeout) {
```

Comment:
Correct signature should be `function checkIfValidTimestamp(totalTimeout: number): boolean {` 
but tests give 
![image](https://user-images.githubusercontent.com/19617248/101848983-c4b52780-3b24-11eb-8202-627b8a4ff7ed.png)